sdk/resourcemanager/Azure.ResourceManager/docs/MigrationGuide.md

```diff
@@ -0,0 +1,287 @@
+# Migrating from old to preview management SDK.
+
+There are several differences between the old sdk and this preview. Here's an example of how to create a Virtual Machine with both SDKs:
+
+## Create a Virtual Machine example
+
+### Import the namespaces
+#### Old (Microsoft.Azure.Management._)
+```C#
+using Microsoft.Azure.Management.Compute;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Rest;
+using System;
+using System.Threading.Tasks;
+```
+#### New (Azure.Azure.ResourceManager._ Preview)
+```C#
+using Azure.Identity;
+using Azure.ResourceManager.Core;
```

Comment:
Isn't the root namespace Azure.ResourceManager now?

Reply:
You're right, thanks for pointing that out

Reply:
Actually, I think it's `Azure.ResourceManager.Resources`